File: example/src/main/java/com/xwray/groupie/example/HeaderItemDecoration.java
package com.xwray.groupie.example;

import android.support.annotation.ColorInt;

public class HeaderItemDecoration extends com.xwray.groupie.example.core.decoration.HeaderItemDecoration {
    public HeaderItemDecoration(@ColorInt int background, int sidePaddingPixels) {
        super(background, sidePaddingPixels, R.layout.item_header);
    }
}

File: example/src/main/java/com/xwray/groupie/example/HeaderItemDecoration.kt
package com.xwray.groupie.example

import androidx.annotation.ColorInt

class HeaderItemDecoration(@ColorInt background: Int, sidePaddingPixels: Int)
    : com.xwray.groupie.example.core.decoration.HeaderItemDecoration(
        background, sidePaddingPixels, R.layout.item_header)
